<launch>
    <arg name="enable_micro" default="true"/>

    <!-- rosserial -->
    <group if="$(arg enable_micro)">
        <node name="microcontroller_node" pkg="rosserial_python" type="serial_node.py" output="screen">
            <param name="port" type="string" value="/dev/ttyACM0" />
            <param name="baud" type="int" value="115200" />
        </node>
    </group>

    <!-- <node name="django_interface_node" pkg="django_interface" type="django_interface_node.py" /> -->

    <node name="status_server_node" pkg="django_interface" type="status_server_node.py" />

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
    
</launch>